tests/FixerConfiguration/FixerConfigurationResolverTest.php

```diff
@@ -152,13 +152,9 @@ public function testResolveWithNormalizers()
             }),
         ));
 
-        $catched = null;
+        $this->expectException('Symfony\Component\OptionsResolver\Exception\InvalidOptionsException');
```

Comment:
Reading the test again, using `assertSame()` was probably the best way actually. Sorry I misled you.

Reply:
No worries :)